src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```diff
@@ -48,6 +56,18 @@ export class ActionBarOverflowMenu {
     }
 
     private countOverflowedItems = () => {
+        if (this.overFlowIcon) {
```

Comment:
The function name `countOverflowedItems` is no more descriptive for what it does. I would suggest this instead `getOverflowMenuTriggerContent` @LucyChyzhova

Reply:
let's simplify it to getOverflowTriggerContent